servers/wanaku-server-quarkus/src/main/java/ai/wanaku/server/quarkus/api/v1/management/targets/TargetsBean.java

```diff
@@ -30,20 +32,43 @@ public class TargetsBean {
     @Inject
     ServiceRegistry serviceRegistry;
 
+    @Inject
+    ServiceRepository serviceRepository;
+
     public void configureTools(String service, String option, String value)
             throws IOException {
-        Map<String, String> configurations = toolsConfigurations(service);
-        configurations.put(option, value);
+        // TODO why is it needed?
+//        Map<String, String> configurations = toolsConfigurations(service);
+//        configurations.put(option, value);
```

Comment:
This is loading the service configurations directly from the downstream services and then persisting them. 

I am not sure if we need to keep all of it. We must, however, persist any configuration that had been previously set. For instance, for the Kafka downstream, if the user had set the `bootstrapHost` configuration, then we want to keep that persisted.